### DISCUSSION TITLE
How do we run DAMAVER with window?

### DISCUSSION THREAD

**Discussion by nanolife1** [biosaxs-com/atsas-community#1928] [Model evaluation and manipulation] [answered]
Hi ATSAS,
Is it possible to run DAMMIN on window to create 10 envelops and then feed these envelops into DAMAEVER and run on window?
Thanks,

**Comment by franke-biosaxs** [comment, marked as answer]
It has always been possible to do this with the scripting capabilities of your respective platform, but with ATSAS-4.0, this became even easier.  If you start the new ATSAS Launcher application, you will see:
<img width="1117" alt="Partial screenshot of ATSAS Launcher" src="https://github.com/user-attachments/assets/e02d17ca-dae7-4b35-8092-e76447a51088">
Actually you will see more, this is only a partial screenshot. Note the DAMMIN button (the greyed out ones are work in progress and will be enabled in later releases). Click the DAMMIN button and a wizard will start. Follow it, select your data and eventually you can select the number of repetitions and whether or not to apply refinement. The individual runs will be executed in parallel on your local computer, all data will be stored in a temporary directory that you then can save wherever you need it.

Comments and feedback about the new launcher application would be highly welcome!

Regards

Daniel

**Reply by nanolife1** [reply]
Hi Daniel,
This sounds exciting. I will download this version and try out.
Thank you very much.

**Reply by nanolife1** [reply]
Hi Daniel,
I have question regarding running DAMMIN with window and linux. From may same data set, one that was ran on automatic setting of DAMMIN Window give a loosen bead model (attached Fig 1 and Fig 1.log). However, when I ran DAMMIN on linux, the bead model is smoother (attached fig 2 and Fig 2.log). Is this possible to adjust the parameters of DAMMIN on Window to get smoother bead model? The reason I ask is because I can't access to linux to run DAMMIN so I want to DAMMIN on Window to work. Or is the loosen bead model on Window good enough to be published?
[Fig 2.log](https://github.com/user-attachments/files/16283505/Fig.2.log)
[Fig 1.log](https://github.com/user-attachments/files/16283506/Fig.1.log)
<img width="148" alt="Fig 1" src="https://github.com/user-attachments/assets/ed14947b-00a5-40f0-a718-9acc0d154414">
<img width="154" alt="Fig 2" src="https://github.com/user-attachments/assets/31ee5617-f213-4077-9884-abd01e35835f">

Thanks.

**Comment by maxim-biosaxs** [comment]
Judging from the log files, both runs are on Windows but one uses dammiN, the other dammiF - that makes the difference.
Having said that, dammin model also looks reasonable

**Reply by nanolife1** [reply]
Sorry, i posted the wrong log file for the Fig 2. Here is the log file of Fig 2. 
[tRNA_SLA11.log](https://github.com/user-attachments/files/16284044/tRNA_SLA11.log)
BTW, do you think the Fig 1 quality is good enough to be published? I always see people published with smooth bead model? so I am a bit worry with loosen bead model.
Thanks,

**Comment by maxim-biosaxs** [comment]
To my mind, the model in Fig. 1 looks compact and not too loose

**Reply by nanolife1** [reply]
Thank you very much for your opinion. I feel more confident with my model.
Best,
